### PR TITLE
Emily docs

### PR DESCRIPTION
**Pull request by AshtonStephens**
## Description

Closes: #1053

## Changes

Adds Emily API doc.

- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: emily/README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+> **Caution:** when deploying to a stage that's deemed to be for an active development stack (`dev`, `local`, `unit-test`) all persistent resources will be set to be deleted when the stack is redeployed or deleted. It's not recommended to use any of these environments for anything you want to be remotely persistent.
+
+### Custom Domains
+
+When you specify a custom domain name for your stack you need to also provide the hosted zone id which you can find in AWS console. You need to have purchased this before attempting to deploy the API to a custom domain. If you don't provide a custom root domain name it won't bother attaching the api to a custom domain, so you're safe if you don't want to purchase one.
+
+The custom domain will generate a different api based on the stage that you deploy to. Lets assume your custom domain is `example.com`
+
+- stage = `prod` -> `example.com`
+- stage = `dev` -> `dev.example.com`
```
Comment (Jiloc) [inline emily/README.md:94]: don't we also append the stage name? i.e. currently dev is hosted on dev.sbtc-emily-dev.com

**Review by Jiloc — APPROVED** [review]
Just a question, everything looks good!

File: emily/README.md (hunk 1 above)
Comment (AshtonStephens) [reply emily/README.md:94]: No, that's just because that's the url I bought, but that was fully manual.

Comment (AshtonStephens) [reply emily/README.md:94]: Mind if I resolve this?

**Pull request merged**